app/models/slate_lineup.py

```diff
@@ -22,7 +22,8 @@ class SlateLineupModel(BaseModel):
 
     @staticmethod
     @xray_recorder.capture_async('models_slate_lineup_get_slate_lineup')
-    async def get_slate_lineup_with_fallback(slate_lineup_id: str, user_id: Optional[str] = None,
+    async def get_slate_lineup_with_fallback(slate_lineup_id: str,
```

Comment:
This is a formatting change. All the arguments were on their own line except for one, so I gave that argument its own separate line.